### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by dheitmueller**
These are a couple of trivial fixes resulting from not reading the instructions when trying out the tree for the first time.

### CODE REVIEW

File: launchEncoderTestPattern.sh [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
 
+if [ -z "$1" ]; then
+    TARGETSERVER="127.0.0.1"
+    echo "Target Server not specified, assuming ${TARGETSERVER}..."
+else
+    TARGETSERVER="$1"
+fi
+
+if [ -z "$2" ]; then
+    TARGETPATH="1234"
```
Comment (codingtmd) [inline launchEncoderTestPattern.sh]: @colleenkhenry , this probably should be called as "STREAMID", right?

File: launchServer.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-go run main.go "/.www" >/dev/null
+go run main.go "./www" >/dev/null
```
Comment (codingtmd) [inline launchServer.sh:3]: @dheitmueller thanks for fixing this typo.

**Review by codingtmd — COMMENTED** [review]
LGTM

File: launchEncoderTestPattern.sh (hunk 1 above)
Comment (dheitmueller) [reply launchEncoderTestPattern.sh]: I can call it whatever you like.  I don't claim to be a DASH expert.

**Review by codingtmd — APPROVED** [review]
Thank you, dude!

**Pull request merged**